server/project.clj

```diff
@@ -4,6 +4,7 @@
   :license {:name "Apache License, Version 2.0"
             :url "https://www.apache.org/licenses/LICENSE-2.0"}
   :dependencies [[org.clojure/clojure "1.9.0-alpha14"]
+                 [org.clojure/core.async "0.3.442"]
```

Comment:
I added it. There are fixes to core.async wrt channel exceptions that are in that build. The release far more often than the main.